src/views/sidebar.js

```diff
@@ -1,14 +1,27 @@
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
 
-import { actionsNav } from '../actions';
+import { actionsNav, actionsInfo } from '../actions';
 import ComponentIcon from '../components/icon';
 import Text from '../components/text';
 import { colors } from '../styles';
 import { View, TouchableOpacity } from 'react-native';
 import store from '../store';
 
+let interval;
+
 class Sidebar extends Component {
+  componentDidMount() {
+    interval = setInterval(() => {
+      actionsInfo.getInfo();
```

Comment:
We already call `getInfo()` in the constructor in `src/actions/info.js`. No need to call it here again. So it looks like `componentDidMount` and `componentWillUnmount` are not needed.